src/platform/testing/unit/mocha-setup.js

```diff
@@ -30,8 +36,23 @@ export default function setupJSDom() {
 
   // Prevent warnings from displaying
   /* eslint-disable no-console */
-  console.error = () => {};
-  console.warn = () => {};
+  if (process.env.LOG_LEVEL === 'debug') {
+    console.error = (error, reactError) => {
+      if (reactError instanceof Error) {
+        console.log(filterStackTrace(reactError.stack));
+      } else if (error instanceof Response) {
+        console.log(`Error ${error.status}: ${error.url}`);
+      } else if (error instanceof Error) {
+        console.log(filterStackTrace(error.stack));
+      } else if (error.includes('The above error occurred')) {
```

Comment:
I notice when I run all the tests with `.env.LOG_LEVEL` set to `debug`. I get some failing tests with the error:
```
TypeError: error.includes is not a function
```
which seems to be from here.

Reply:
Ah, I added this line after I had done most of my testing, I can tweak this.